routers/user/home.go

```diff
@@ -322,6 +322,10 @@ var issueReposQueryPattern = regexp.MustCompile(`^\[\d+(,\d+)*,?\]$`)
 
 // Issues render the user issues page
 func Issues(ctx *context.Context) {
+
+	// Distinguish Issues from Pulls. (Q: Why not make that distinction during routing and call different functions?)
```

Comment:
For this one I think you'd be right - there's not much sense issues and pulls being the same route (at least not anymore)